src/pe_reports/helpers/bulletin/bulletin_generator.py

```diff
@@ -0,0 +1,274 @@
+  INTEL_ITEM_ID                     Cyber Sixgill intel item id
+"""
+
+# Standard Python Libraries
+import datetime
+import logging
+import os
+
+# Third-Party Libraries
+from docopt import docopt
+import jinja2
+import pandas as pd
+import pdfkit
+
+# cisagov Libraries
+from pe_reports.data.db_query import connect
+from pe_source.data.pe_db.config import cybersix_token
+from pe_source.data.sixgill.api import intel_post
+
+LOGGER = logging.getLogger(__name__)
+
+
+def get_post(id):
+    """Retrieve a cybersix post based on the intel item id."""
+    query = f"_id:{id}"
+    token = cybersix_token()
+    resp = intel_post(token, query, frm=0, scroll=False, result_size=1)
+    return resp
+
+
+def date_format(value, format="%m/%d/%Y"):
+    """Format a date field."""
+    val = datetime.datetime.strptime(value, "%Y-%m-%dT%H:%M:%S")
+    return val.strftime(format)
+
+
+def html_builder(text):
+    """Build out an html string for command line usage."""
+    input_type = input(
+        "Which of the following would you like to insert:\n [P]aragraph\n [B]ulleted List \n [N]umbered List\n Please provide a selection:"
+    )
+    if input_type == "P":
+        LOGGER.info("Paragraph Selected")
+        paragraph = input("Please enter paragraph text:")
+        paragraph = f"<p> {paragraph} </p>"
+        text = text + f"\n {paragraph}"
+
+    elif input_type == "B":
+        LOGGER.info("Bulleted List Selected. Enter [D] when done.")
+        bullets = "<ul>\n"
+        while True:
+            item = input("Enter line item: ")
+            if item == "D":
+                bullets = bullets + "</ul>"
+                break
+            bullets = bullets + f"<li>{item}</li>\n"
+        text = text + f"\n {bullets}"
+
+    elif input_type == "N":
+        LOGGER.info("Numbered List Selected")
+        bullets = "<ol>\n"
+        while True:
+            item = input("Enter line item: ")
+            if item == "D":
+                bullets = bullets + "</ol>"
+                break
+            bullets = bullets + f"<li>{item}</li>\n"
+        text = text + f"\n {bullets}"
+    else:
+        LOGGER.info("Invalid Selection")
+
+    cont = input("Would you like to add more content (Y/N): ")
+    if cont == "Y":
+        text = html_builder(text)
+
+    return text
+
+
+bulletin_path = os.path.dirname(os.path.realpath(__file__))
+
+
+def generate_cybersix_bulletin(
+    id,
+    user_text="",
+    output_directory="/var/www/cybersix_bulletins",
+    filename="_Bulletin.pdf",
+):
+    """Generate a bulletin based on a provided cybersix id."""
+    if not filename.endswith(".pdf"):
+        filename = filename + ".pdf"
+
+    templateLoader = jinja2.FileSystemLoader(searchpath=bulletin_path)
+    templateEnv = jinja2.Environment(loader=templateLoader, autoescape=True)
+    templateEnv.filters["date_format"] = date_format
+    TEMPLATE_FILE = "bulletin_template.html"
+    template = templateEnv.get_template(TEMPLATE_FILE)
```

Comment:
Stick with snake case, instead of slipping into camel case.  It's always preferable to stick with one type of casing throughout:
```suggestion
    template_loader = jinja2.FileSystemLoader(searchpath=bulletin_path)
    template_env = jinja2.Environment(loader=template_loader, autoescape=True)
    template_env.filters["date_format"] = date_format
    TEMPLATE_FILE = "bulletin_template.html"
    template = template_env.get_template(TEMPLATE_FILE)
```